etna/search/views.py

```diff
@@ -403,30 +409,14 @@ def get_api_aggregations(self) -> List[str]:
         Called by `get_api_kwargs()` to get a value to include as 'aggregations'
         in the API request.
 
-        In the API response, the items with the highest number of matches are
-        included for each aggregation. Those values are used to indicate
-        counts for each 'bucket', and to update the form field choices, so that
-        the most relevant filter options are shown.
-        """
-        values = []
-        for aggregation in (
-            Aggregation.COLLECTION,
-            Aggregation.LEVEL,
-            Aggregation.TOPIC,
-            Aggregation.CLOSURE,
-            Aggregation.HELD_BY,
-            Aggregation.CATALOGUE_SOURCE,
-            Aggregation.GROUP,
-            Aggregation.TYPE,
-        ):
-            item_count = 10
-            if aggregation == Aggregation.GROUP:
-                # Fetch more 'groups' so that we receive a counts
-                # for any bucket/tab options we might be showing
-                # (not just the 10 most popular)
-                item_count = 30
-            values.append(f"{aggregation}:{item_count}")
-        return values
+        The aggregations params may be specific to a bucket and will be filtered upon.
+        Returns a list of aggregation params.
+        Ex: ["group:30", "catalogue:10",]
+        """
+        for bucket in self.bucket_list:
+            if bucket.is_current:
+                return bucket.api_aggregations_params
+        return []
```

Comment:
I feel like knowing the 'current bucket' is common enough that we should add a `current_bucket` cached property to the view to pull this out. I have a feeling it could be useful do be able to do `view.current_bucket` in templates too.

Reply:
Made use of `BucketList.current`